contracts/cw1-whitelist/src/contract.rs

```diff
@@ -0,0 +1,198 @@
+use anyhow::Error;
```

Comment:
Don't use `anyhow` on production, its a bit more overhead than `thiserror` - better switch to the second one. It also gives you better error testability.